### ISSUE TITLE
Return original note ID on "cannot create note because it is a duplicate" error

### ISSUE THREAD

**Issue by Idane** [FooSoft/anki-connect#214]
Use case: I would like to upsert notes without having to find the duplicate myself

**Comment by Idane** [comment]
@FooSoft Is this something you would consider desirable for the project? 
If so, I could take a crack at it.

**Comment by FooSoft** [comment]
Yeah, I think that should be fine. We'd probably want this new behavior behind an optional flag, just so that we don't change the behavior of existing tools that use this API.
